.github/workflows/mobile-i18n-autofill-pr.yml

```
@@ -0,0 +1,89 @@
+      - name: Check for changes
+        id: check_changes
+        run: |
+          if git status --porcelain cmp-android/ feature/ | grep -q .; then
+            echo "has_changes=true" >> $GITHUB_OUTPUT
+          else
+            echo "has_changes=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Commit and Push changes
+        if: steps.check_changes.outputs.has_changes == 'true'
+        run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)